test/Parlot.Tests/Parlot.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net5.0;netcoreapp3.1;net6.0</TargetFrameworks>
+    <TargetFrameworks>net5.0;netcoreapp3.1;net6.0;net7.0</TargetFrameworks>
```

Comment:
NET 5 is obsolete, netcoreapp3.1 will be in couple of weeks, maybe drop them?

Reply:
I don't think so, I'd rather just add an ignore warnings for these because I can't just drop it, end-users will get the same warning and decide to update their apps instead, when they want. I'll remove them but after a while.

Reply:
This comment is for tests, so my comment is not that accurate, I will still keep these for the same reason though.

Reply:
Cool. I think that it's hard to break tests between runtimes unless you are testing the conditional compilation. MS is dull in that sense about backwards compatibility.... But It doesn't take that long to run them.